README.md

```diff
@@ -38,174 +38,42 @@ or
-
-WMR supports Rollup plugins, and there's a growing list of [**configurations and recipes**](https://github.com/preactjs/wmr/wiki/Configuration-Recipes) in the wiki, including:
-
-- [minify the HTML output](https://github.com/preactjs/wmr/wiki/Configuration-Recipes#minifying-html)
-- [import directories of modules](https://github.com/preactjs/wmr/wiki/Configuration-Recipes#importing-directories-of-files)
-- [implement filesystem-based routing](https://github.com/preactjs/wmr/wiki/Configuration-Recipes#filesystem-based-routing--page-component-loading)
-- [add a service worker](https://github.com/preactjs/wmr/wiki/Configuration-Recipes#service-worker)
+| Package                                                    | Description                                                       | Version                                                                                                                                              |
+| ---------------------------------------------------------- | :---------------------------------------------------------------- | :--------------------------------------------------------------------------------------------------------------------------------------------------- |
+| [wmr](packages/wmr)                                        | The tiny all-in-one development tool for modern web apps          | [![wmr npm](https://img.shields.io/npm/v/wmr.svg)](https://npm.im/wmr)                                                                               |
+| [create-wmr](packages/create-wmr)                          | Create a new WMR project in seconds                               | [![create-wmr npm](https://img.shields.io/npm/v/create-wmr.svg)](https://npm.im/create-wmr)                                                          |
+| [@wmr-plugins/directory-import](packages/directory-plugin) | Allows you to import multiple files in a single statement         | [![@wmr-plugins/directory-import npm](https://img.shields.io/npm/v/@wmr-plugins/directory-import.svg)](https://npm.im/@wmr-plugins/directory-import) |
+| [@wmr-plugins/nomodule](packages/nomodule-plugin)          | Allows you to create legacy bundles alongside WMR's modern output | [![@wmr-plugins/nomodule npm](https://img.shields.io/npm/v/@wmr-plugins/nomodule.svg)](https://npm.im/@wmr-plugins/nomodule)                         |
+| [@wmr-plugins/service-worker](packages/sw-plugin)          | Allows you to add a service worker                                | [![@wmr-plugins/service-worker npm](https://img.shields.io/npm/v/@wmr-plugins/service-worker.svg)](https://npm.im/@wmr-plugins/service-worker)       |
+| [preact-iso](packages/preact-iso)                          | Isomorphic async tools for Preact                                 | [![preact-iso npm](https://img.shields.io/npm/v/preact-iso.svg)](https://npm.im/preact-iso)                                                          |
```

Comment:
Let me know if the verbiage could be improved. Some I could copy/paste, but I'm not entirely sure `directory-plugin` in particular is super clear.

Reply:
```suggestion
| Package | Description | Version
| --- | :--- | :--- |
| [wmr](packages/wmr) | Tiny all-in-one development tool for modern web apps | [![wmr npm](https://img.shields.io/npm/v/wmr.svg)](https://npm.im/wmr)
| [create-wmr](packages/create-wmr) | Create a new WMR project in seconds | [![create-wmr npm](https://img.shields.io/npm/v/create-wmr.svg)](https://npm.im/create-wmr)
| [@wmr-plugins/directory-import](packages/directory-plugin) | Import a directory's files as an Array | [![@wmr-plugins/directory-import npm](https://img.shields.io/npm/v/@wmr-plugins/directory-import.svg)](https://npm.im/@wmr-plugins/directory-import)
| [@wmr-plugins/nomodule](packages/nomodule-plugin) | Generate legacy fallback bundles for older browsers | [![@wmr-plugins/nomodule npm](https://img.shields.io/npm/v/@wmr-plugins/nomodule.svg)](https://npm.im/@wmr-plugins/nomodule)
| [@wmr-plugins/service-worker](packages/sw-plugin) | Bundle service workers | [![@wmr-plugins/service-worker npm](https://img.shields.io/npm/v/@wmr-plugins/service-worker.svg)](https://npm.im/@wmr-plugins/service-worker)
| [preact-iso](packages/preact-iso) | Optimal code-splitting, hydration and routing for Preact | [![preact-iso npm](https://img.shields.io/npm/v/preact-iso.svg)](https://npm.im/preact-iso)
```